ddtrace/ext/tags.go

```diff
@@ -87,12 +87,18 @@ const (
 	// ErrorType specifies the error type.
 	ErrorType = "error.type"
 
-	// ErrorStack specifies the stack dump.
+	// ErrorStack specifies the stack dump when the error is thrown.
 	ErrorStack = "error.stack"
 
+	// ErrorHandlingStack specifies the stack dump when the error is captured.
+	ErrorHandlingStack = "error.handling_stack"
```

Comment:
We are adding these two new tags, what are the implications on the ingestion side? Especially for error tracking?

Reply:
The new tags will change the behavior of error tags on spans; `handling_stack` should already be handled from the error tracking side, but we just haven't been taking advantage of it earlier. The fingerprints of spans will change after we release this PR, so we should give them a heads-up before so that they can adjust in the backend.